component/core/src/main/java/org/exoplatform/social/core/jpa/storage/dao/jpa/SpaceDAO.java

```diff
@@ -389,6 +388,11 @@ private void buildPredicates(XSpaceFilter spaceFilter,
       parameterNames.add(PARAM_KEYWORD);
     }
 
+    if (StringUtils.isNotBlank(spaceFilter.getRemoteId())) {
```

Comment:
The test `isLastAccess(spaceFilter)` is missing, isn't it ?
```suggestion
    if (isLastAccess(spaceFilter) && StringUtils.isNotBlank(spaceFilter.getRemoteId())) {
```

Reply:
yes!